fmt

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
Let's remove this.  Even if we wanted to reformat en masse, it would only be useful once.

Reply:
Done.